htdocs/GetCSV.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * Controls access to a module's SQL tables, exporting a CSV file of the table
+ * to user. This script should ensure that only files relative to module's path
+ * specified are accessible.
+ * By calling new NDB_Client(), it also makes sure that the user is logged in to
+ * Loris.
+ *
+ * It also does validation to make sure required config settings are specified.
+ *
+ * Used by "new" style modules.
+ *
+ * PHP Version 5
+ *
+ *  @category Loris
+ *  @package  Loris
+ *  @author   Jordan Stirling <jordan.stirling@mail.mcgill.ca>
+ *  @license  Loris license
+ *  @link     https://github.com/aces/Loris-Trunk
+ */
+
+set_include_path(
+    get_include_path() . ":" .
+    __DIR__ . "/../project/libraries:" .
+    __DIR__ . "/../php/libraries"
+);
+require_once __DIR__ . "/../vendor/autoload.php";
+// Ensures the user is logged in, and parses the config file.
+require_once "NDB_Client.class.inc";
+$client = new NDB_Client();
+$client->initialize("../project/config.xml");
+
+// Checks that config settings are set
+$config =& NDB_Config::singleton();
+$paths  = $config->getSetting('paths');
+
+// Basic config validation
+$basePath = $paths['base'];
+if (empty($basePath)) {
+    error_log("ERROR: Config settings are missing");
+    header("HTTP/1.1 500 Internal Server Error");
+    exit(1);
+}
+
+
+// Now get the file and do file validation
+$TestName = isset($_REQUEST['test_name']) ? $_REQUEST['test_name'] : 'dashboard';
+$subtest  = isset($_REQUEST['subtest']) ? $_REQUEST['subtest'] : '';
+
+if (empty($TestName)) {
+    error_log("Missing required parameters for request");
+    header("HTTP/1.1 400 Bad Request");
+    exit(2);
+}
+
+$list     = load($TestName, $subtest);
+$csv_data = arrayToCSV($list);
+print_r($csv_data);
```

Comment:
print_r won't be in csv format, it'll include array(..).

The easiest way to handle csv formatting that I know of in PHP is to use fputcsv with a "file" of STDOUT.